docs/en/changes/changes.md

```diff
@@ -45,6 +45,7 @@
   - NGINX -> K8S_SERVICE
   - APISIX -> K8S_SERVICE
   - GENERAL -> APISIX
+* Support Apache RocketMQ server monitoring.
```

Comment:
@wankai123 Do we have a guidelines for service hierarchy between general service(virtual MQ) and RocketMQ?

@peachisai Are names from two layers associated? One is from span peer and the other is your service name.

Reply:
There are no hierarchy guidelines for the specific layer. If the service names (virtual MQ and RocketMQ) could be mapped through `hierarchy-definition.yml` then they can build the service hierarchy relationship.

Reply:
@peachisai As we can't merge this due to lack of RocketMQ image, could we try to resolve this? Startinh from comparing the existing RocketMQ span peer and your service name?

About service hierarchy, this is a good doc to refer to, https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/service-hierarchy/

Reply:
Excuse me , I just now have a quick read, so I need to build the service relationship between `virtual MQ` and
`rocketmq server` ?

Reply:
I think this PR is nearly complete, and you can prepare the [Showcase](https://github.com/apache/skywalking-showcase) and the service hierarchy relationship.

you can refer to the others first.

Reply:
Regarding the showcase, we expect to have default setup in.the OAP. 

Although we will setup showcase foe this monitoring, it won't have RocketMQ as Virtual MQ. So, this hierarchy may not be able to show due to general agent shoecase can't have all conjecture services. 

Still we could provide default configurations to make them out of box when users have that deployment. So far, we prefer this works when agents and RocketMQ server run on k8s.

Reply:
@wankai123 Make sense?

Reply:
Good point. Let's support and verify both.